### DISCUSSION TITLE
Gigahorse CPU is stopping after the 3rd table

### DISCUSSION THREAD

**Discussion by lordmidocrypto50** [madMAx43v3r/chia-gigahorse#214] [General]
Hey everyone, 
Is there a problem with the Gigahorse CPU plotting? It keeps stopping in table #4.

**Comment by Kruger1981** [comment]
How much RAM you have? And what C level of plot are you creating?

**Comment by Kruger1981** [comment]
Actually... what are you using to plot? Windows or Linux? SSD, M.2, HDDs in raid? Where are final plots stored? Same drive?

**Comment by lordmidocrypto50** [comment]
Hello, 

Thank you for your reply.

I am sorry for not providing enough details. I am using Windows 10 and I am testing plotting on my PC. The PC has Ryzen 5 5600X and 16 GB Ram. The compression level is C7 and C9. This was done on the CPU plotting. 

I think I figured out what was the issue. I think it was the SSD. It got full suddenly and crashed. I used a 1TB - alarger HDD- and it worked fine.
